src/index.js

```diff
@@ -134,24 +141,34 @@ export default function(options = {}) {
 
               if (getExport) {
                 codeExportDefault = getExport(result.opts.from)
-                Object.keys(codeExportDefault).forEach(k => {
-                  const camelCasedKey = dashesCamelCase(k)
-                  if (reserved.check(camelCasedKey)) {
-                    console.warn(
-                      chalk.yellow('You are using a reserved keyword'),
-                      chalk.cyan(camelCasedKey),
-                      chalk.yellow(
-                        "as className so it's not available in named exports"
+
+                if (escapeDashes) {
+                  const escapedExportDefault = {}
+                  Object.keys(codeExportDefault).forEach(k => {
+                    const camelCasedKey = dashesCamelCase(k)
+                    escapedExportDefault[camelCasedKey] = codeExportDefault[k]
+                  })
+                  codeExportDefault = escapedExportDefault
+                }
+                if (exportNamed) {
```

Comment:
That's true. If you want named exports and you have dashes in the names, you'll need to have `escapeDashes: true`.
However if there are no dashes in the class names, this combination is valid. Should we document this more clearly or disable the combination?